displays/cavity_display/utils/faults.csv

```diff
@@ -47,6 +47,8 @@ PJT,PRL phase standard deviation out of tolerance,In-use PRL phase STD > 0.02 de
 PJT,PRL phase standard deviation out of tolerance,In-use PRL phase STD > 0.02 degrees for 1.3  GHz systems (0.06 degrees for 3.9 GHz systems),Contact LLRF team,RACK,,EDM,$EDM/llrf/rf_srf_wf_prl_rack.edl,"SELTAB=8,SELCHAR=3,FWD=FWD,REV=REV,PRLDESC=PRL Readout,ACQFILE=rf_srf_wf_prl_acq_cm",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,PRLJITSUM,,1,1,PRL phase out of tolerance,
 PZF,Piezo feedback off,Piezo Feedback is not on,Run auto setup,CAV,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=3,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PZT:MODECTRL,1,,2,Piezo feedback off,
 FBS,Piezo detune and cavity feedback not regulating,This alarms if the detune measurement used by the piezo feedback is < -50 Hz or > 50 Hz and the cavity amplitude or phase measurement is out of tolerance.,Contact LLRF team,CAV,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,FBSTATSUM,,2,2,Piezo detune and cavity feedback not regulating,
+PZL,Piezo near limit Alarm,Piezo tuner is at its limit,Run auto setup,CAV,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=3,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PZT:INTEG_AT_LIM.SEVR,,2,2,Piezo tuner is at its limit,

```

Comment:
For the PZL alarm, is it accurate in the long description to say "Piezo tuner is at its limit?" I archived and probed the general PV without the .SEVR attribute and it looks like it only has three states (fine, warning HIGH, and alarm HIHI). Want to check in case I'm making too much of an assumption here though

Reply:
Ya I think so!